src/scripts/components/ErrorMessage.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import IndexHeader from 'components/header/IndexHeader';
+
+class ErrorMessage extends React.Component {
```

Comment:
'class' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).